---
title: "DupLoss-2: Improved Phylogenomic Species Tree Inference under Gene Duplication and Loss"
collection: publications
category: manuscripts
excerpt: 'Accurate species tree reconstruction in the presence of widespread gene duplication and loss is a challenging problem in eukaryote phylogenomics. Many phylogenomics methods have been developed over the years to address this challenge; these range from older methods based on gene tree parsimony to newer quartet-based methods. In this work, we introduce improved software for gene tree parsimony-based species tree reconstruction under gene duplication and loss. The new software, DupLoss-2, uses an improved procedure for computing gene losses and is far more accurate and much easier to use than its previous version released over a decade ago. We thoroughly evaluate DupLoss-2 and eight other existing methods, including ASTRAL-Pro, ASTRAL-Pro 2, DISCO-ASTRAL, DISCO-ASTRID, FastMulRFS, and SpeciesRax, using existing benchmarking data and find that DupLoss-2 outperforms all other methods on most of the datasets. It delivers an average of almost 30% reduction in reconstruction error compared to iGTP-Duploss, the previous version of this software, and a 10% reduction compared to the best performing existing method. DupLoss-2 is written in C++ and is freely available open-source.'
date: 2009-10-01
venue: 'BioRxiv'
paperurl: 'https://www.biorxiv.org/content/10.1101/2024.09.05.611565v1'
citation: 'Parsons, Rachel and Bansal, Mukul. (2024). &quot;DupLoss-2: Improved Phylogenomic Species Tree Inference under Gene Duplication and Loss.&quot; <i>BioRxiv</i>.'
---

The contents above will be part of a list of publications, if the user clicks the link for the publication than the contents of section will be rendered as a full page, allowing you to provide more information about the paper for the reader. When publications are displayed as a single page, the contents of the above "citation" field will automatically be included below this section in a smaller font.
